### ISSUE TITLE
Any ideas how we can avoid rewriting the whole rep for every deploy?

### ISSUE THREAD

**Issue by cggaurav** [axitkhurana/buster#83]

**Comment by joshterrill** [comment]
Yes. Once you have already done one commit and have your site published on github, this is what you have to do (this assumes that your generated files from `buster generate` are in a folder called `/static/` and you are currently in that folder and currently have a local ghost instance running)
1. Type `cd ..` to get out of the static folder
2. Then type `buster generate --domain=http://localhost:3000` to get the updated files into the `static/` folder
3. Type `cd static/` to go back into the folder
4. `git add .` to add the new files to git
5. `git commit -m "new commit"`
6. `git push -u origin master`

This will only upload the changed files to github.
